src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -756,13 +756,36 @@ std::string ContextAttribute::render
 *        the code paths of the rendering process
 *
 */
-std::string ContextAttribute::toJson(bool isLastElement, const std::string& renderMode, RequestType requestType)
+std::string ContextAttribute::toJson(bool isLastElement, const std::string& renderMode, NotificationFormat notifyFormat, RequestType requestType)
 {
   std::string  out;
+  bool         uniqueValues = false;
+  std::string  rMode        = renderMode;  // renderMode is 'const' and cannot be modified
+
+  //
+  // FIXME PR: To be discussed during PR-review
+  //   'notifyFormat' overrides 'renderMode' if NGSI_V2_KEYVALUES or NGSI_V2_VALUES but 'uniqueValues' must be saved
```

Comment:
No worries, we still have time to discuss this.
Postponed for next PR and added to 'postponed list' in issue #1875